src/Adapter/Configuration/LogsConfiguration.php

```diff
@@ -62,6 +70,17 @@ public function getConfiguration()
     public function updateConfiguration(array $configuration)
     {
         if ($this->validateConfiguration($configuration)) {
+            $checkEmails = explode(',', $configuration['logs_email_receivers']);
+            $errors = [];
+            foreach ($checkEmails as $email) {
+                if (!Validate::isEmail($email)) {
```

Comment:
@okom3pom Could you inject in construct the Validate Adapter ? Thanks

Reply:
@Progi1984 isEmail is missing in the Validate Adapter my PR target 1.7.8 and normaly the code is freeze 

The PR of @PululuK ( https://github.com/PrestaShop/PrestaShop/pull/24622 ) add new fonctionality and he target the dev branch

Reply:
I can close mine and you validate @PululuK PR  in 1.7.8? Because it's a regression ?

Reply:
Hi @okom3pom, you can copy past this part in my PR ;)

Reply:
Good idea @PululuK (and you will rebase your PR after the merge of this one). #modeCoop